src/test/java/md/ramaiana/foodmarket/dao/OrderGoodDaoTest.java

```diff
@@ -0,0 +1,96 @@
+package md.ramaiana.foodmarket.dao;
+
+
+import md.ramaiana.foodmarket.config.DataJdbcConfig;
+import md.ramaiana.foodmarket.model.Client;
+import md.ramaiana.foodmarket.model.Good;
+import md.ramaiana.foodmarket.model.Order;
+import md.ramaiana.foodmarket.model.OrderGood;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.data.jdbc.DataJdbcTest;
+import org.springframework.context.annotation.Import;
+
+import java.time.OffsetDateTime;
+
+@DataJdbcTest
+@Import(DataJdbcConfig.class)
+class OrderGoodDaoTest {
+    @Autowired
+    private OrderGoodDao orderGoodDao;
+
+    @Autowired
+    private OrderDao orderDao;
+
+    @Autowired
+    private ClientDao clientDao;
+
+    @Autowired
+    private GoodDao goodDao;
+
+    @Test
+    void test_create() {
+        Order someOrder = someOrder();
+        Good someGood = someGood();
+        orderGoodDao.save(OrderGood.builder()
+                .orderId(someOrder.getId())
+                .goodId(someGood.getId())
+                .quantity(1000f)
+                .sum(10000f)
+                .weight(50f)
+                .build());
+    }
+
+    @Test
+    void test_read() {
+        Good existingGood = someGood();
+        orderGoodDao.existsById(existingGood.getId());
+    }
+
+    @Test
+    void test_update() {
+        Good someGood = someGood();
+        goodDao.save(someGood);
```

Comment:
`someGood` is saved in method `someGood()`. What is necessity of calling `dao.save` again?